### ISSUE TITLE
Cash flow statement does not work if there are no profit & loss expenses

### ISSUE THREAD

**Issue by hanselke** [frappe/erpnext#9098]
when selling items with no stock, cashflow statement does not show net profit & loss line under operating cashflow.

This is due to net_profit_loss = get_net_profit_loss(income, expense, period_list, filters.company) breaking when there are no expenses.

minor bug

**Comment by tundebabzy** [comment]
Hello @hanselke 
Please add screen shots for quicker resolution
